spec/support/email.rb

```diff
@@ -1,4 +1,8 @@
 RSpec.configure do |config|
   config.include(EmailSpec::Helpers)
   config.include(EmailSpec::Matchers)
+
+  config.before(:each) do
+    ActionMailer::Base.deliveries.clear
+  end
```

Comment:
Started doing this but quickly realised its a far bigger job - most controller specs make assertions about emails sent and none of them use the email_spec helpers.

Refactoring all of them would be a PR in itself